apps/libc/dynamic/armv7/contextc.c

```diff
@@ -138,7 +138,7 @@ Return Value:
     // Context.
     //
 
-    StackTop = (PVOID)(Context->uc_stack.ss_sp + Context->uc_stack.ss_size -
+    StackTop = (PVOID)((char *)Context->uc_stack.ss_sp + Context->uc_stack.ss_size -
```

Comment:
These aren't ideal. In general I try to use typedef types, so these would need to be PCHAR rather than char *

However I think maybe we should try a different approach instead of adding more casting everywhere. Let's create some macros in include/minoca/lib/types.h, something to the effect of:
```
#define POINTER_ADD(_Base, _Addition) \
    (PVOID)((UINTN)(_Base) + (UINTN)(_Addition))

```
... and equivalent for subtraction. Then we can use them instead. And at least in this case, it would save us a cast, since we could remove the cast here to (PVOID).